docs/book/advanced-guide/practical/secrets-management.md

```diff
@@ -105,13 +245,48 @@ their values.
 * Setting it to `SECRET_AND_KEY_EXISTS` (the default) validates both the secret existence as
 well as the existence of the exact key-value pair.
 
+
+{% hint style="warning" %}
+If you have secrets registered through both the [centralized secrets management](#centralized-secrets-store)
+and [a secrets manager](#secrets-management-with-secrets-managers), ZenML will
+first try to fetch the secret from the centralized secrets management and only
+fall back to the secrets manager if the secret is not found. This means that if
+you have a secret registered with the same name in both centralized secrets
+store 
```

Comment:
I think this thought needs completing?

Reply:
Thanks for noticing, this got away from me.